### PR TITLE
Replace getOrNil with get function in documentation

### LINKED ISSUE #1267: Document enhanced get function
With https://github.com/roboll/helmfile/pull/513 merged, we can use `get` function to accomplish most (or everything) of what would required to use `getOrNil` before.

Currently there is no documentation for it. See https://github.com/roboll/helmfile/issues/465#issuecomment-562109298

### PR DESCRIPTION
**Pull request by renehernandez**
Fix #1267

### CODE REVIEW

File: docs/writing-helmfile.md [hunk 1]
```diff
@@ -14,18 +14,18 @@ That is, the following example let `helmfile` fail when you have no `eventApi.re
...
 
-In case it isn't a mistake and you do want to allow missing keys, use the `getOrNil` template function:
+In case it isn't a mistake and you do want to allow missing keys, use the `get` template function:
 
 ```
-{{ .Values | getOrNil "eventApi.replicas" }}
+{{ .Values | get "eventApi.replicas" }}
 ```
 
 This result in printing `<no value` in your template, that may or may not result in a failure.
```
Comment (renehernandez) [inline docs/writing-helmfile.md:23]: @mumoshu Is this still true if using `get` instead of `getOrNil`?

**Comment by renehernandez** [comment]
@mumoshu Any update on this?

**Comment by renehernandez** [comment]
@mumoshu Is there anything else required to get this in?

File: docs/writing-helmfile.md (hunk 1 above)
Comment (mumoshu) [reply docs/writing-helmfile.md:23]: No. `{{ .Values | get "eventApi.replicas" }}` fails with an error when the value does not exist.

So I guess we'd better change the example above to `{{ .Values | get "eventApi.replicas" nil }}`

**Comment by mumoshu** [comment]
@renehernandez Sorry for the delay! This looks almost good. If you could address the only comment I made here https://github.com/roboll/helmfile/pull/1268#discussion_r436202436, I'll merge this shortly.

Thanks for your patience and contribution!

File: docs/writing-helmfile.md (hunk 1 above)
Comment (renehernandez) [reply docs/writing-helmfile.md:23]: Done

**Pull request merged**